draft-ietf-quic-ack-frequency.md

```diff
@@ -206,13 +206,14 @@ ACK-eliciting threshold:
   received. A value of 0 is invalid. Receipt of an invalid value MUST be
   treated as a connection error of type FRAME_ENCODING_ERROR.
 
-Update Max Ack Delay:
+Request Max Ack Delay:
 
-: A variable-length integer representing an update to the peer's `max_ack_delay`
-  transport parameter (Section 18.2 of {{QUIC-TRANSPORT}}). The value of this
-  field is in microseconds. Any value smaller than the `min_ack_delay`
-  advertised by this endpoint is invalid. Receipt of an invalid value MUST be
-  treated as a connection error of type PROTOCOL_VIOLATION.
+: A variable-length integer representing a request to update to the peer's
+  `max_ack_delay` transport parameter (Section 18.2 of {{QUIC-TRANSPORT}}).
```

Comment:
Suggestion: "A variable-length integer representing the value to which the endpoint requests the peer to update its `max_ack_delay` (Section 18.2 of {{QUIC-TRANSPORT}})."

Reply:
I went with your suggestion with one less 'to'. It's definitely more correct, but I'd like to find something easier to read.

I came up with the following, but I'm not sure it's better:
"A variable-length integer representing the updated max_ack_delay (Section 18.2 of {{QUIC-TRANSPORT}}) value the endpoint is requesting of the peer."